docs/Configuration/agent-configuration.md

```diff
@@ -441,6 +441,25 @@ agent_options:
           etc:
             - /etc/%%
 ```
+
+## Script execution timeout
+
+The `script_execution_timeout` allows you to change the default script execution timeout.
+
+- Optional setting (integer)
+- Default value: 300
+- Maximum value: 3600
+- Config file format:
+  ```yaml
+  agent_options:
+    config:
+      options:
+        distributed_interval: 3
+        distributed_tls_max_attempts: 3
+        logger_tls_period: 10
```

Comment:
```suggestion
    config:
```

Maybe we cut all the keys that don't correspond w/ script execution timeout? To make the example clearer (less words). It looks like we could make this same change to other examples in the agent options docs.

Reply:
I agree. I committed a suggestion.

Reply:
Looks good! Can you please make this same change for the other examples on this doc page? I think it should be quick.